### PR TITLE
Chore/pandas boosting

### PR DESCRIPTION
**Pull request by matthieu-boussard**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - Add new boosting decision routes.
-
+- Add boosting decision route to pandas client.
```
Comment (Giska) [inline CHANGELOG.md:13]:
```suggestion
- Add boosting decision route to pandas client.

```

File: pyproject.toml [hunk 2]
```diff
@@ -35,7 +35,7 @@ flake8 = "^3.7.9"
 taskipy = "^1.1.3"
 
 [tool.taskipy.tasks]
-test = "poetry run pytest --durations=10 tests/*.py"
+test = "poetry run pytest --durations=10 tests/test_pandas.py::TestPandasBoostingGeneratorWithOperation"
```
Comment (Giska) [inline pyproject.toml:38]:
```suggestion
test = "poetry run pytest --durations=10 tests/*.py"
```

File: tests/test_pandas.py [hunk 3]
```diff
@@ -538,3 +544,72 @@ def test_get_generator_decision_tree_with_pdtimestamp(self):
         self.assertNotEqual(decision_tree.get("configuration"), None)
         self.assertNotEqual(decision_tree.get("trees"), None)
         self.assertEqual(decision_tree, ground_truth_decision_tree)
+
+
```
Comment (Giska) [inline tests/test_pandas.py:548]:
```suggestion
```

**Review by Giska — CHANGES_REQUESTED** [review]

**Review by Giska — APPROVED** [review]

File: tests/test_pandas.py (hunk 3 above)
Comment (matthieu-boussard) [reply tests/test_pandas.py:548]: lint doesn't agree

**Pull request merged**